app/scripts/components/RegistrationFilters/RegistrationFilters.tsx

```diff
@@ -0,0 +1,283 @@
+              >
+                <option value="">-Any-</option>
+                <option value="full">Full/Overpaid</option>
+                <option value="partial">Partial</option>
+                <option value="full-partial">Full/Partial</option>
+                <option value="over">Overpaid</option>
+              </select>
+            </div>
+          )}
+          {showFilter('filterRegType') && (
+            <div className="col-md-2 col-sm-6 stacked-spacing-col-sm">
+              <label>Registrant type:</label>
+              <select
+                className="form-control"
+                ng-model="queryParameters.filterRegType"
+                ng-options="r.id as r.name for r in conference.registrantTypes"
+              >
+                <option value="">-Any-</option>
+              </select>
+            </div>
+          )}
+          {showFilter('filterAccountTransfersByPaymentType') && (
+            <div className="col-md-2 col-sm-6 stacked-spacing-col-sm">
+              <label>Payment type:</label>
+              <select
+                className="form-control"
+                value={queryParameters.filterAccountTransfersByPaymentType}
+                onChange={(event) =>
+                  setQueryParam(
+                    'filterAccountTransfersByPaymentType',
+                    event.target.value,
+                  )
+                }
+              >
+                <option value="">-Any-</option>
+                <option value="ACCOUNT_TRANSFER">Account Transfer</option>
+                <option value="SCHOLARSHIP">Scholarship</option>
+              </select>
+            </div>
+          )}
+          {showFilter('filterAccountTransfersByExpenseType') && (
+            <div className="col-md-2 col-sm-6 stacked-spacing-col-sm">
+              <label>Expense type:</label>
+              <select
+                className="form-control"
+                value={queryParameters.filterAccountTransfersByExpenseType}
+                onChange={(event) =>
+                  setQueryParam(
+                    'filterAccountTransfersByExpenseType',
+                    event.target.value,
+                  )
+                }
+              >
+                <option value="">-Any-</option>
+                <option value="REGISTRATION">Registration</option>
+                <option value="MISCELLANEOUS_ITEM">Misc. Expense</option>
+                <option value="CHILDCARE">Childcare</option>
+                <option value="STAFF_TAXABLE_ITEM">
+                  Staff Taxable Expense
+                </option>
+              </select>
+            </div>
+          )}
+          <div className="more-filters col-md-2 col-sm-6 text-right stacked-spacing-col-md">
+            <br />
+            <Button
+              className="btn btn-default"
+              active={showMoreFilters}
+              onClick={() => setShowMoreFilters(!showMoreFilters)}
+            >
+              Filters{' '}
```

Comment:
You could add styles to the `<i>` is you need space between the text and the icon.